src/components/AuthLayout/_auth-layout.scss

```diff
@@ -0,0 +1,49 @@
+@import '../../globals/scss/vars';
+@import '../../globals/scss/colors';
+@import '../../globals/scss/typography';
+@import '../../globals/scss/spacing';
+@import '../../globals/scss/helper-mixins';
+@import '../../globals/scss/import-once';
+@import '../../globals/scss/css--reset';
+
+@include exports('auth-layout') {
+    .#{$prefix}--auth-wrapper{
+        width: 100%;
+        height: 100vh;
+        background-color:var(--ui-01);
+        display: flex;
+        flex-direction: row;
+
+        .#{$prefix}--auth-background-wrapper{
+            width: 60%;
+            border-right: 1px solid var(--ui-03);
+            @include breakpointMax(bp--sm--major) {
+                display: none;
+            }
+
+            
+        }
+
+        .#{$prefix}--auth-background-content{
```

Comment:
We need to add a scrollbar to the the content if it is higher than `100vh`. The image could be `position fixed`.

Reply:
Will use `position fixed` then

Reply:
Will use `position fixed` then